toolchain/BUILD

```diff
@@ -0,0 +1,83 @@
+load(":cc_toolchain_config.bzl", "cc_toolchain_config")
+package(default_visibility = ["//visibility:public"])
```

Comment:
ultra nit: add space between load and package.

Reply:
Done!